nss-day-cohort-48/bangazon-tenntom
# duplicate lineitems

Change to profile.py on the action of cart to show only lineitems on the profile view

Changes:

When a client requests the /profile/cart resource, the response incorrectly included both lineitems and line_items with the same objects. This fix removes the line_items array.

Made changes to the profile.py  file on the cart action. 

Steps to test:
Git fetch --all
Pull branch tlc-lineitems
From the command line, run ./seed_data.sh
start the server python3 manage.py runserver
In Postman, paste this key into the Authorization header. Token 9ba45f09651c5b0c404f37a2d2572c026c146694
From Postman, request profile . http://localhost:8000/profile/cart
Confirm that information is returned for order with an id of 8 for customer 5 and includes a key for lineitems, as one word, with an entry in it for Rio with a product id of 21. 
Check the code differences and see if anything looks crazy.

Related Issues
Fixes Bug:  #24
